src/tasks/Directory.Build.targets

```diff
@@ -12,8 +12,14 @@
 
   <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net462'))">
     <!-- These assemblies and their dependencies are made available by MSBuild on .NET Framework, we only reference for compilation -->
-    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataToolsetVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
-    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonToolsetVersion)" ExcludeAssets="Runtime" PrivateAssets="All"  NoWarn="NU1901;NU1902;NU1903;NU1904" />
+    <PackageDownloadAndReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesToolsetVersion)" Folder="lib/net462" />
+    <PackageDownloadAndReference Include="System.Buffers" Version="$(SystemBuffersToolsetVersion)" Folder="lib/net461" />
```

Comment:
we're referencing lib/net461 here but check for net462 above, is that expected?

Reply:
Yes the ref assembly should be referenced here.

Reply:
Same for System.Memory and System.Threading.Tasks.Extensions.